src/bootstrap/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to bootstrap will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
```

Comment:
I believe we should have a section here for unreleased changes?

Reply:
There is no such thing as unreleased changes, because there are no releases. Part of the point of nagging you to read the changelog is that there's no way to opt-out of breaking changes.

Reply:
> Changes that do not affect contributors to the compiler or users
> building rustc from source don't need an update to `VERSION` (although they
> should still update the `CHANGELOG`).

Oh, I meant where to update the `CHANGELOG` if there is no breaking changes?

Reply:
Ah good point. Maybe they should just be added to the old version? That breaks the assumption that versions are immutable ... I'll add an `unreleased` section like you mentioned originally.